Sources/Clappr/Classes/Base/Core.swift

```diff
@@ -64,9 +64,9 @@ open class Core: UIBaseObject, UIGestureRecognizerDelegate {
 
         #if os(iOS)
         addTapRecognizer()
-        #endif
-
         bindEventListeners()
```

Comment:
Here we've added the macro `#if os(iOS)` but is always present inside `bindEventListeners`. We can keep just one to make it simple. :)